torch/nn/modules/distance.py

```diff
@@ -18,8 +18,8 @@ class PairwiseDistance(Module):
         keepdim (bool, optional): Determines whether or not to keep the vector dimension.
             Default: False
     Shape:
-        - Input1: :math:`(N, D)` where `D = vector dimension`
-        - Input2: :math:`(N, D)`, same shape as the Input1
+        - Input1: :math:`(N, D)` or :math:`(D)` where `D = vector dimension` and `N = batch dimension`
+        - Input2: :math:`(N, D)` or :math:`(D)`, same shape as the Input1
```

Comment:
How do you think it should be phrased?

```
- Output: :math:`(N)` or :math:`()` as per the Input shape. If :attr:`keepdim` is ``True``, then :math:`(N, 1) or :math:`(1)`.
```

Sorry I always get flustered with phrasing these things.
Thanks!

Reply:
Hm, I think the way you phrased it is good! Just one minor thing, second shape in the first part should be `()` instead of `(D)`:
```
Output: :math:`(N)` or :math:`()` based on input dimension.
```

Reply:
Ah. Right 😅 Thanks!